torchvision/ops/boxes.py

```diff
@@ -297,3 +297,36 @@ def generalized_box_iou(boxes1: Tensor, boxes2: Tensor) -> Tensor:
     areai = whi[:, :, 0] * whi[:, :, 1]
 
     return iou - (areai - union) / areai
+
+
+def masks_to_boxes(masks: torch.Tensor) -> torch.Tensor:
+    """
+    Compute the bounding boxes around the provided masks
+
+    Returns a [N, 4] tensors, with the boxes in (X, Y, X, Y) format.
```

Comment:
nit: Probably instead of (X, Y, X, Y) you can write
```
 Both sets of boxes are expected to be in ``(x1, y1, x2, y2)`` format with
``0 <= x1 < x2`` and ``0 <= y1 < y2
```
as this is written in other docstrings in ops.